connector_job_subscribe/models/res_users.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Cédric Pigeon
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    subscribe_job = fields.Boolean('Jobs Notifications',
+                                   default=True)
```

Comment:
Could you add a tooltip explaining that job notifications are sent only when this is active AND the user is a connector manager?